### ISSUE TITLE
[Feature Request] Report Categories

### ISSUE THREAD

**Issue by ghost** [sethsec/celerystalk#69]
Hello, first of all thank you for a creating this tool, I am sure a lot of work must have been put into it. I have tried few scanners around, they all have their purpouse but celerystalk seems to be most extensible, user friendly and adding user defined commands is a doodle and most importantly quick!.

Also it would be really beneficial to have a report that generates categories (http, https, smb or by port number 80, 443, 445 etc) instead/alongside? of displaying the output from each tool individually. It would make the report much easier to gather data from.

Thanks once again for your hard work.

**Comment by rafaleon69** [comment]
Sorry for a long reply yeah that's the plan to filter the output by the services. Sth like:
http
   --nikto
   --nmap http
   --CMSMap eg.

https
   --nikto
   --nmap http
   --CMSMap eg.

ssh
  --ssh-user-enum
  --Nmap--ssh etc.
  
It would make the output more manageable to go through.

**Comment by WilliamCXVI** [comment]
Hey rafaleon69, I know this is unrelated, but did you ever manage to get the Huawei p9 firmware to support packet injection on Nougat? I have another android device (not huawei p9, but lg g5 h830) that likewise is in the same situation as yours:: supporting monitor mode, but no packet injection. 

In any case, I may install Oreo if it really comes down to that. Also, I know you might be confused as to think h830 is actually bcm43455, but that actually isn't true as my device had already contained 4358 chip.

**Comment by rafaleon69** [comment]
@WilliamCXVI I eventually did manage. It was a long time ago, from my experience back then, Oreo Nethunter did not work really well, problems with a terminal etc. What I did, was to install Nougat, put a Nethunter on it and copy wifi drivers from Oreo as this one supports packet injection (7_112_300_14_sta). Then I could only manage to call tools using LD_PRELOAD. But I cannot give you any more details as I moved on to other device.
